java/src/jmri/implementation/DefaultTransit.java

```diff
@@ -572,6 +572,32 @@ public boolean removeLastTemporarySection(Section s) {
 
     }
 
+    private TransitType transitType = TransitType.USERDEFINED;
+
+    /**
+     * Set Transit Type.
+     * <ul>
+     * <li>USERDEFINED - Default Save all the information.
+     * <li>DYNAMICADHOC - Created on an as required basis, not to be saved.
+     * </ul>
+     * @param type constant of section type.
+     */
```

Comment:
It's recommended to instead use:
`/** {@inheritDoc} */`

This way, the JavaDoc for the sub class always gets the same JavaDoc as the super class for this method.